src/xc/lda_c_vwn.jl

```diff
@@ -2,13 +2,17 @@
 VWN5 LDA correlation according to Vosko, Wilk, and Nusair, (DOI 10.1139/p80-159).
 """
 function energy_per_particle(::Val{:lda_c_vwn}, ρ)
+    if ρ <= 1e-8
```

Comment:
That's a little large. Can you do 1e-14 or 1e-16? We should also put this into the primal (with a tolerance depending on the floating-point type).

Reply:
sure, but it already is the primal? we could of course put it one level higher

Reply:
Sorry by primal I meant `master`. yes putting it one level higher is better, I think.